exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -90,6 +90,9 @@ func (e *tracesExporter) pushTraceData(ctx context.Context, td ptrace.Traces) er
 				serviceName = v.Str()
 			}
 			for j := 0; j < spans.ScopeSpans().Len(); j++ {
+				resAttr[conventions.AttributeOtelScopeName] = spans.ScopeSpans().At(j).Scope().Name()
+				resAttr[conventions.AttributeOtelScopeVersion] = spans.ScopeSpans().At(j).Scope().Version()
```

Comment:
I just realized that we are violating OTel schema's representation that has been followed so far in the clickhouse table schema. Ideally, we need to add new columns for these fields. @Frapschen, will this be too big of a breaking change for such an addition? If that's the case, span attributes would be a better place. A scope cannot be mapped 1:1 to a resource. One resource can have multiple instrumentation scopes.

Reply:
I prefer add them to span attribute directly.